backend/src/main/java/io/castles/core/model/GameSettingsDTO.java

```diff
@@ -0,0 +1,31 @@
+package io.castles.core.model;
+
+import io.castles.core.tile.Tile;
+import io.castles.game.GameSettings;
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Value
+@EqualsAndHashCode
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+public class GameSettingsDTO {
+    int turnTimeSeconds;
+    int maxPlayers;
+    List<Long> tileList;
```

Comment:
Why dont you return a list of `TileDTO`s here?

Reply:
No real reason, other than LobbySettingsDTO does it the same. Feel free to change if that helps